scripts/us_census/acs5yr/subject_tables/common/generate_col_map_test.py

```diff
@@ -12,12 +12,22 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 """Tests for generateColMap."""
-import os
+
 import csv
 import json
+import os
+import sys
 import unittest
 
-from .generate_col_map import generate_stat_var_map, process_zip_file
+# Allows the following module imports to work when running as a script
+# relative to data/scripts/
+sys.path.append(
+    os.path.sep.join([
+        '..' for x in filter(lambda x: x == os.path.sep,
+                             os.path.abspath(__file__).split('scripts/')[1])
+    ]))
+
+from generate_col_map import generate_stat_var_map, process_zip_file
 
```

Comment:
Similar to the previous comment, we may want to check if we need to replace `generate_col_map` in line 30 with `us_census.acs5yr.subject_tables.common.generate_col_map`, to resolve the ModuleNotFoundError in cloudbuild?

Reply:
reverting this change.